invenio_communities/communities/schema.py

```diff
@@ -55,26 +55,25 @@ class CommunityAccessSchema(Schema):
     ]))
 
 
+# TODO this schema is duplicated from invenio-rdm-records, can't be imported
```

Comment:
open issue: should be moved to `invenio-vocabularies`, named `VocabularyRelationSchema`

Reply:
Created issue: https://github.com/inveniosoftware/invenio-vocabularies/issues/188